ColabTool/Shared/Post/index.jsx

```diff
@@ -3,8 +3,8 @@ import {
   Card,
 } from '@bufferapp/components';
 import style from './style.css';
-import PostButtonPanel from '../PostButtonPanel';
 import PostDetails from '../PostDetails';
+import PostTopDetails from '../PostTopDetails';
```

Comment:
There are only two hard things in Computer Science: naming things, cache invalidation and off-by-one errors.